src/main/rx/crsf.c

```diff
@@ -112,6 +112,15 @@ struct crsfPayloadRcChannelsPacked_s {
 
 typedef struct crsfPayloadRcChannelsPacked_s crsfPayloadRcChannelsPacked_t;
 
+STATIC_UNIT_TESTED uint8_t crsfFrameCRC(void)
```

Comment:
It would be more elegant (and reusable) if this operated a pointer to the crsfFrame that's passed in.

Reply:
I didn't change this function,.. just moved it up in the code.  That can be changed to accept the pointer instead if you want.

Reply:
Can be done in another PR then.